rezerwacje-app/src/main/java/org/niewidoczniakademicy/rezerwacje/service/RoomService.java

```diff
@@ -64,6 +65,21 @@ public GetRoomResponse saveRoom(final AddRoomRequest request) {
                 .build();
     }
 
+    public GetRoomResponse editRoom(EditRoomRequest request) {
+        Room room = roomRepository.findById(request.getId())
+                .orElseThrow(() -> new RoomNotFoundException("Room with id " + request.getId() + " not found"));
+
+        room.setBuilding(request.getBuilding());
+        room.setName(request.getName());
+        room.setCapacity(request.getCapacity());
+        roomRepository.save(room);
```

Comment:
I would reassign room here, it's good practise to keep all ids up to date after saves. I mean something like that.
`room = roomRepository.save(room); `

Reply:
Done